### ISSUE TITLE
List misp-object fields in alphabetical order

### ISSUE THREAD

**Issue by VVX7** [MISP/MISP#5584]
**Is your feature request related to a problem? Please describe.**
misp-object fields are randomly ordered when loaded in the UI.  It is annoying searching for fields in objects.

**Describe the solution you'd like**
List misp-object fields in alphabetical order.

**Comment by adulau** [comment]
Indeed, it's a good point. We will have a look to implement finally the UI priority value along with the alphabetical order. If you can get a free license for a game, @iglocska will do it right away.

**Comment by iglocska** [comment]
Have all of theirs already anyway, but will have a look indeed. ;)

**Comment by VVX7** [comment]
I've already earmarked some Ubisoft swag for you but you'll need to come to Montreal to pick it up ;)
